src/backend/distributed/utils/citus_outfuncs.c

```diff
@@ -81,9 +87,15 @@
 					 booltostr(node->fldname))
 
 /* Write a character-string (possibly NULL) field */
+#if (PG_VERSION_NUM >= 90600)
+#define WRITE_STRING_FIELD(fldname) \
+	(appendStringInfo(str, " :" CppAsString(fldname) " "), \
+	 outToken(str, node->fldname))
+#else
 #define WRITE_STRING_FIELD(fldname) \
 	(appendStringInfo(str, " :" CppAsString(fldname) " "), \
 	 _outToken(str, node->fldname))
+#endif
```

Comment:
Hm, can't we get rid of that distinction? Why don't we just have our copy of _outToken as outToken() for 9.5?